src/Fields/Input/Input.js

```diff
@@ -41,6 +47,7 @@ const Input = ({ id, type, initial, ...props }) => {
             field: { id, initial, type, ...props }
           })
         }
+        required
```

Comment:
This parameter should depend from error which we are have in our state. Also we need a pattern either for just showing the validation mssg.